### DISCUSSION TITLE
Is it necessary for the editor to support the shell format?

### DISCUSSION THREAD

**Discussion by Hongjiuhuiji** [massCodeIO/massCode#461] [Q&A] [answered]
I often need to use Linux and AWS in my personal work, and there are many shell scripts that need to be recorded in daily life, but I found that there is no shell in the format supported by MassCode software, so I am very distressed.

**Comment by antonreshetov** [comment, marked as answer]
You can use Bash for these purposes.

With the full list of supported grammars, you can familiarize yourself [here](https://github.com/massCodeIO/massCode/tree/master/src/renderer/components/editor).

**Reply by Hongjiuhuiji** [reply]
Thanks for your help. I am seeking the link you provided. Hope the software gets better and better.
